app/views/lti_deployment/_new_course.html.erb

```diff
@@ -0,0 +1,10 @@
+<p>
+  <%= form_with local: true, url: lti_deployment_new_course_path do |f| %>
+  <%= f.hidden_field :display_name, :value => session[:lti_course_name] %>
```

Comment:
Use the newer hash syntax `value: session[:lti_course_name]`, and below.

Also please fix the indentation of this code